mllib/src/main/scala/org/apache/spark/ml/tuning/ParamGridBuilder.scala

```diff
@@ -20,13 +20,14 @@ package org.apache.spark.ml.tuning
 import scala.annotation.varargs
 import scala.collection.mutable
 
-import org.apache.spark.annotation.Experimental
+import org.apache.spark.annotation.{Experimental, Since}
 import org.apache.spark.ml.param._
 
 /**
  * :: Experimental ::
  * Builder for a param grid used in grid search-based model selection.
  */
+@Since("1.2.0")
 @Experimental
 class ParamGridBuilder {
```

Comment:
Can you add a since tag for the constructor?  (You may need to check generating the docs to make sure this works.)